src/azure-cli/azure/cli/command_modules/backup/custom.py

```diff
@@ -113,6 +115,20 @@ def list_associated_items_for_policy(client, resource_group_name, vault_name, na
 
 def set_policy(client, resource_group_name, vault_name, policy):
     policy_object = _get_policy_from_json(client, policy)
+    retention_range_in_days = policy_object.properties.instant_rp_retention_range_in_days
+    schedule_run_frequency = policy_object.properties.schedule_policy.schedule_run_frequency
+
+    # Validating range of days input
+    if schedule_run_frequency == 'Weekly' and retention_range_in_days != 5:
```

Comment:
Have you show up these requirements for retention_range_in_days in your help message?

Reply:
@pvrk

Reply:
@Juliehzl: These parameters are required only for Azure VM backup and not Azure File backup. Hence for this PR, they are not required and I will consider this separately.

Reply:
Can you provide a document link to let users know how to set policy correctly in your help message?